onboarding-sdk/onboarding-sdk-product/README.md

```diff
@@ -0,0 +1,154 @@
+# Onboarding SDK Product
+
+This library has been developed to provide a set of Java utility classes to simplify the work of handle **Selfcare Product** as string.
```

Comment:
when do we have such a need?

Reply:
Selfcare Products is a collection of PagoPA products available for use by institutions. Each product contains specific information, such as its status, admitted role, or a filepath template for building contract necessary for selfcare business logic.

Currently, these product records are stored in a MongoDB collection which are accessible through a REST API. 
We are planning to replace this architecture with a file-based approach in our roadmap. This change is driven by the fact that all product information requires only 36KB of storage so we will saving in terms of resource and maintenance. 

The Onboarding SDK Product offers a set of classes designed for managing this collection of records using a file.